.dagger-ci/Taskfile.yml

```diff
@@ -27,3 +28,70 @@ tasks:
     dir: daggerci
     cmds:
       - python -m pytest --cov --cov-report=term-missing --cov-report=html --log-cli-level NOTSET --show-capture no --log-cli-level=INFO --runslow {{.CLI_ARGS}}
+
+  setup-python-venv:
+    desc: Setup python virtual environment
+    internal: true
+    cmds:
+      - python -m venv {{.PYTHON_VENV}}
+      - bash -c "
```

Comment:
Are those multiline cmds working without `\` at the end or prefixing it with the yaml `|` syntax? Looks like GitHubs syntax highlighter does not like it

Reply:
It does seem to work ... works on my machine xD

Reply:
Yeah, I added the `|` character. GitHubs syntax highlighter should be happier now :)